src/WinUIEx/HwndExtensions.cs

```diff
@@ -115,6 +115,161 @@ public static void CenterOnScreen(IntPtr hwnd, double? width = null, double? hei
             SetWindowPosOrThrow(new HWND(hwnd), new HWND(), left, top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
         }
 
+        /// <summary>
+        /// Moves the window to the specified coordinates
+        /// </summary>
+        /// <param name="hwnd">Window handle</param>
+        /// <param name="x">Left side of the window</param>
+        /// <param name="y">Top side of the window</param>
+        public static void MoveToPoint(IntPtr hwnd, double x, double y)
+        {
+            PInvoke.GetWindowRect(new HWND(hwnd), out RECT windowRect);
+            var w = windowRect.right - windowRect.left;
+            var h = windowRect.bottom - windowRect.top;
+            var left = x;
+            var top = y;
+            SetWindowPosOrThrow(new HWND(hwnd), new HWND(), (int)left, (int)top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="hwnd"></param>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        public static void CenterOnPoint(IntPtr hwnd, double x, double y, double? width = null, double? height = null)
```

Comment:
I gave it a separate name because of its separate behaviour.  MoveTo moves the nearest corner/side to the Placement, and only the centre when it is sent to the centre.  CentreOnPoint always moves the centre of the **window** to the coordinates.  

Placement.MiddleCentre moves it to the window to centre of the **screen**